config/jest.base.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Note there are two base jest configurations.  One for unit tests and one for integration tests.